doc/cephadm/troubleshooting.rst

```diff
@@ -272,21 +268,21 @@ To call miscellaneous like ``ceph-objectstore-tool`` or
     election_strategy: 1
     0: [v2:127.0.0.1:3300/0,v1:127.0.0.1:6789/0] mon.myhostname
 
-This command sets up the environment in a way that is suitable
-for extended daemon maintenance and running the daemon interactively. 
+The cephadm shell sets up the environment in a way that is suitable
+for extended daemon maintenance and running daemons interactively. 
 
 .. _cephadm-restore-quorum:
 
-Restoring the MON quorum
+Restoring the Monitor Quorum
 ------------------------
```

Comment:
Here and at 301, the title underlines are too short. That's causing the Pr-docs build to fail.

Reply:
I was hoping that this wasn't the case, and we could save a few bits by instituting a new short-title-underline standard. Alas.

Reply:
Mostly-unrelated, but similarly irritating: Admonitions in ordered lists in RST cannot be indented if you want them to build correctly in Sphinx. They just have to run on and on, in one giant line.  I'm not sure how I figured this out, but I know that it involved hours of trial and error. As far as I can tell, this RST-Sphinx behavior is not documented anywhere except in my occasional online complaints.

Reply:
"Note", "Warning", and "Important" boxes are called "admonitions". The "warning" box (the yellowish one) at the top of this page is an admonition: https://docs.ceph.com/en/quincy/